packages/docs/components/molecules/Accordion/index.md

```diff
@@ -9,3 +9,42 @@
 
 - [Examples](./examples.html)
 - [Twig API](./twig-api.html)
+
+## Usage
+
+Once the [package installed](/guide/installation/), simply include the template in your project:
+
+```js{2,8}
+import { Base, createApp } from '@studiometa/js-toolkit';
+import { Accordion } from '@studiometa/ui';
+
+class App extends Base {
+  static config = {
+    name: 'App',
+    components: {
+      Accordion,
+    },
+  };
+}
+
+export default createApp(App, document.body);
+```
+
+```twig{16}
+{% set items = [
```

Comment:
**question** Should we have a distinct section like `required props` for this type of component who do nothing if not provided with data ?

Reply:
That could be interesting. Maybe a `*` symbol in the title could do the trick?

Reply:
Can we do that in a future PR ?

Reply:
sure, this is not blocking